coilsnake/util/common/helper.py

```diff
@@ -54,3 +54,8 @@ def lower_if_str(x):
 
 def grouped(iterable, n):
     return izip(*[iter(iterable)]*n)
+
```

Comment:
He just added a newline to the end of the file (editors like that for some reason..)